tests/Export/ProductServiceTest.php

```diff
@@ -117,29 +117,53 @@ public function testFindsVariantForInactiveProduct(): void
         $this->assertSame('FINDOLOGIC VARIANT', $product->getName());
     }
 
-    public function testIfMoreThanOneVariantExistsItWillUseVariantInformationInsteadOfMainProductInformation(): void
+    public function variantProvider(): array
     {
+        return [
+            'variant information is used instead of product information' => [
+                'mainProductActive' => false,
+                'firstVariantActive' => true,
```

Comment:
`firstVariantActive` may suit this use case, but it is more flexible when you just provide the variants as data. This way it is easier adding new data to the data provider.

```php
$parentId = Uuid::randomHex();

// ...
[
    'variants' => [
        [
            'active' => true,
            'id' => $parentId
            // ...
        ],
        [
            'active' => false,
            'parentId' => $parentId
            // ...
        ]
    ],
    'expectedChildCount' => 1',
];
```